internal/databases/greenplum/backup_fetch_handler_test.go

```diff
@@ -50,3 +51,131 @@ func TestPrepareContentIDsToFetch(t *testing.T) {
 		assert.Equal(t, tc.contentIDsToFetch, contentIDsToFetch)
 	}
 }
+
+func TestBuildFetchCommand(t *testing.T) {
+	beforeValue := config.CfgFile
+	defer func() {
+		config.CfgFile = beforeValue
+	}()
+	config.CfgFile = "testConfig"
```

Comment:
I agree this is tricky. But `buildFetchCommand` relies on this global variable. May be there is some other way to do it. 

Without this setting `cmdLine` would be something like this `... --config= ...`. I think it's not good.